Utils/Dataflow/pyDKB/common/misc.py

```diff
@@ -0,0 +1,48 @@
+"""
+pyDKB.common.misc
+
+Miscellanious utility functions.
+"""
+
+import sys
+import inspect
+from datetime import datetime
+
+from types import logLevel
+
+# Datetime format for log messages
+DTFORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def log(message, level=logLevel.INFO, *args):
+    """ Output log message with given log level.
+
+    :param message: message to output
+    :type message: str
+    :param level: log level of the message
+    :type level: ``pyDKB.common.types.logLevel`` member
+    :param *args: additional prefixes (will be output between log
+                  level prefix and message body)
+    :type *args: str
+    """
+    if not logLevel.hasMember(level):
+        self.log("Unknown log level: %s" % level, logLevel.WARN)
+        level = logLevel.INFO
+    if type(message) == list:
+        lines = message
+    else:
+        lines = message.splitlines()
+    if args:
+        prefix = ' ' + ' '.join(['(%s)' % p for p in args])
+    else:
+        frm = inspect.stack()[1]
+        mod = inspect.getmodule(frm[0])
+        prefix = ' (%s)' % mod.__name__
```

Comment:
This can cause errors if `__name__` is missing, for example when calling the function from interactive python interpreter:

    [vaulov@aiatlas171 Dataflow]$ python
    Python 2.7.5 (default, Jun 20 2019, 20:27:34)
    [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
    Type "help", "copyright", "credits" or "license" for more information.
    >>> from pyDKB.common.misc import log
    2019-08-01 16:21:06 (WARN) (pyDKB.dataflow.cds) Submodule failed (No module named invenio_client.contrib)
    >>> log('123')
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "pyDKB/common/misc.py", line 40, in log
        prefix = ' (%s)' % mod.__name__
    AttributeError: 'NoneType' object has no attribute '__name__'

Reply:
Fixed, thank you.